backend/src/main/kotlin/org/loculus/backend/service/files/FilesTable.kt

```diff
@@ -0,0 +1,17 @@
+package org.loculus.backend.service.files
+
+import org.jetbrains.exposed.sql.Table
+import org.jetbrains.exposed.sql.kotlin.datetime.datetime
+import java.util.*
```

Comment:
another wildcard snuck in, maybe an IntelliJ setting?

Reply:
thanks, yes, I think I fixed it now.